### ISSUE TITLE
Restore backup admin settings 

### ISSUE THREAD

**Issue by testuser2015** [RocketChat/Rocket.Chat#2337]
Your Rocket.Chat version: (make sure you are running the latest)
I want to restore the Rocketchat admin settings.   I want to take backup of existing settings and to restore it to updated Rocket chat version.

**Comment by engelgabriel** [comment]
If you are using a MongoDB server, you dont lose the settings between updates.
If you want to backup your data, you can use the `mongoexport` command.

**Comment by KonradTUM** [comment]
Is it possible that someone explains a bit more on how this works?
I need to do this for my rocket.chat server. I needed to reinstall and now i need all the layout settings?

I am running the server under Ubuntu 16.04 with snap. 
On my old rocket.chat server I did `sudo snap run rocketchat-server.backupdb`
and copied the backup to the new one and imported like it is explained in the docs `sudo /snap/rocketchat-server/current/bin/mongorestore --db parties \
~/backup_data/var/snap/rocketchat-server/<version>/dump/parties/`

But the layout data is missing, can anyone help?
